src/components/AnnotationTarget/AnnotationTarget.tsx

```diff
@@ -1,18 +1,36 @@
 import * as React from 'react';
 import classNames from 'classnames';
 import noop from 'lodash/noop';
+import scrollIntoView, { Options } from 'scroll-into-view-if-needed';
 import { KEYS } from 'box-ui-elements/es/constants';
 import './AnnotationTarget.scss';
 
 type Props = {
     annotationId: string;
     children: React.ReactNode;
     className?: string;
+    isActive?: boolean;
     onSelect?: (annotationId: string) => void;
 };
 
+const scrollOptions: Options = {
+    behavior: 'smooth',
+    block: 'center',
+    inline: 'center',
+    scrollMode: 'if-needed',
+};
+
 const AnnotationTarget = (props: Props, ref: React.Ref<HTMLAnchorElement>): JSX.Element => {
-    const { annotationId, children, className, onSelect = noop, ...rest } = props;
+    const { annotationId, children, className, isActive, onSelect = noop, ...rest } = props;
+    const innerRef = React.useRef<HTMLAnchorElement>(null);
+
+    React.useImperativeHandle(ref, () => innerRef.current as HTMLAnchorElement, [innerRef]);
```

Comment:
is the type cast here necessary?

Reply:
The correct type wasn't getting picked up so for this didn't figure it would be a big deal to typecast it.